cl/opinion_page/forms.py

```diff
@@ -401,45 +465,41 @@ def clean_pdf_upload(self) -> bytes:
             )
         return pdf_data
 
-    def make_panel(self) -> None:
-        if self.pk == "tennworkcompapp":
-            self.cleaned_data["panel"] = list(
-                filter(
-                    None,
-                    [
-                        self.cleaned_data["lead_author"],
-                        self.cleaned_data.get("second_judge"),
-                        self.cleaned_data.get("third_judge"),
-                    ],
-                )
-            )
-        elif self.pk == "me":
-            self.cleaned_data["panel"] = self.cleaned_data["panel"]
-        else:
-            self.cleaned_data["panel"] = [self.cleaned_data["lead_author"]]
-
     def make_item_dict(self) -> None:
         """Make item dictionary for adding to our DB
 
         :return: None
         """
         lead_author = self.cleaned_data.get("lead_author")
+
+        judges_list = [
+            j.name_full for j in self.cleaned_data.get("panel", []) if j
+        ]
+        if self.cleaned_data.get("judges"):
+            # We can have additional judges from charfield that are not in the system
+            judges_list.append(self.cleaned_data.get("judges"))
+
         self.cleaned_data["item"] = {
             "source": Docket.DIRECT_INPUT,
             "cluster_source": SOURCES.DIRECT_COURT_INPUT,
+            "cluster_disposition": self.cleaned_data.get("disposition", ""),
```

Comment:
the language here is inconsistent really with how we use it.  

I'm not sure why you are using cluster_disposition instead of just disposition 

same with date_filed.

Reply:
yes, i already checked carefully, i saw that the only model that has the disposition field is OpinionCluster, so it is only "disposition"

regarding date_filed, it has been defined as item["case_dates"] since 2019, making the change would imply modifying the code where the make_objects function is used, that could be done in an independent PR